pages/docs/concepts/channel.md

```diff
@@ -19,4 +19,4 @@ graph LR
     C --> E[Consumer]
     C --> F[Consumer]
 ```
-The diagram above shows the communication between a `producer` and `consumer`, with the producer sending a `message` through the `channel`. The channel then queues the message to the specific consumer.
+xx
```

Comment:
why is there a change to the channel page? and why are you deleting this and adding 2 XX's? 

is this a mistake?